frontend/src/features.ts

```diff
@@ -0,0 +1,73 @@
+export interface Feature {
+  name: string;
+  description: string;
+  active: boolean;
+}
+
+export const features: Feature[] = [
```

Comment:
nit: does this need to be exported?

Reply:
Good point! I have removed those exports.